packages/push/src/relation.js

```diff
@@ -4,14 +4,28 @@ export default class Relation {
     this.resource = resource;
   }
 
-  create(attributes) {
-    let resource = new this.resource(attributes);
+  push(attributes) {
+    return this.client.post(this.endpoint, this.normalize(attributes));
+  }
+
+  create() {
+    return this.push(...arguments);
+  }
 
-    return this.client.post(this.endpoint, resource);
+  update() {
+    return this.push(...arguments);
   }
 
-  delete(identifier) {
-    return this.client.delete(`${this.endpoint}/${identifier}`);
+  delete(attributes) {
+    return this.client.delete(this.endpoint, this.normalize(attributes));
+  }
+
+  normalize(attributes) {
```

Comment:
I think we should do this on the resource level so stuff like `new User("user-1")` works as well.